compiler/rustc_target/src/callconv/x86_win64.rs

```diff
@@ -25,6 +25,7 @@ where
                 // FIXME(eddyb) there should be a size cap here
                 // (probably what clang calls "illegal vectors").
             }
+            BackendRepr::ScalableVector { .. } => unreachable!("scalable vectors are unsupported"),
```

Comment:
I can guess why you used `panic` in one case and `unreachable` in the other, but I'd prefer it was consistent in either direction.
```suggestion
            BackendRepr::ScalableVector { .. } => panic!("scalable vectors are unsupported"),
```